lib/std/simd.zig

```diff
@@ -0,0 +1,395 @@
+//! This module provides functions for working conveniently with SIMD (Single Instruction; Multiple Data),
+//! which may offer a potential boost in performance on some targets by performing the same operations on
+//! multiple elements at once.
+//! Please be aware that some functions are known to not work on MIPS.
+
+const std = @import("std");
+const builtin = @import("builtin");
+
+pub const Vector = std.meta.Vector;
+
```

Comment:
I tend to explain the basic data structures as regular comment, so that first users know how to deal with stuff
and to not clutter the top level decl with information.
```zig
// A Vector is continuous and has comptime-known length and element sizes with the following layout:
// data  A B C D
// index 0 1 2 3
// -> right
// <- left
```

Reply:
This seems like the responsibility of the language docs, since SIMD vectors are a language-provided feature. Am I wrong?

Reply:
good catch. Can you add it to the section `Vectors` in the langref or create an issue for this?

Reply:
They already seem to be doing fine, to me.

Reply:
I think it is fine like this and my stuff belongs into the specification.